package.json

```
@@ -2,38 +2,52 @@
   "name": "web-streams-polyfill",
   "version": "3.1.0",
   "description": "Web Streams, based on the WHATWG spec reference implementation",
-  "main": "dist/polyfill",
-  "browser": "dist/polyfill.min.js",
+  "main": "dist/polyfill.js",
+  "browser": "dist/polyfill.js",
   "module": "dist/polyfill.mjs",
   "types": "dist/types/polyfill.d.ts",
-  "typesVersions": {
-    ">=3.6": {
-      "dist/types/*": [
-        "dist/types/ts3.6/*"
-      ]
-    }
+  "exports": {
+    ".": {
+      "import": "./dist/polyfill.mjs",
+      "require": "./dist/polyfill.js"
+    },
```

```suggestion
    ".": {
      "import": "./dist/ponyfill.es6.mjs",
      "require": "./dist/ponyfill.es6.js"
    },
```

I'd argue these are better default entry points.

---

Makes sense. I'll flip them around: ES6 becomes the default, and ES5 can be accessed through `import from 'web-streams-polyfill/es5'`.

---

I understand the reasoning for making the ponyfill the default. But then I'm not entirely sure what to call the polyfilling variant (i.e. the one that *does* mutate `globalThis`). `import from "web-streams-polyfill/polyfill"` sounds a bit silly. 😛